docs/source/kedro_project_setup/settings.md

```diff
@@ -0,0 +1,98 @@
+# Project settings
+
+A Kedro project's `settings.py` file contains the settings for the project, such as library component registration, custom hooks registration, etc. This document explains how settings work and which settings are available.
+
+## Default settings
+A Kedro settings file doesn’t have to define any settings if it doesn’t need to. Each setting has a sensible default value. These defaults live in the module `src/<project_name>/settings.py`.
```

Comment:
I'd just include this in the introduction rather than its own section.
```suggestion
A project does not _need_ to have a settings file – in fact, by default, all code in `settings.py` is commented out. In the case that settings are not supplied, Kedro chooses sensible default values. You only need to edit `settings.py` if you wish to change to values other than the defaults.
```